Changes

```diff
@@ -1,6 +1,8 @@
 {{$NEXT}}
           die on Experian error, instead of silently ignoring them
 
+2.51      2018-04-20 09:42:39+00:00 UTC
```

Comment:
I don't think this line should be here? Next version would be 2.53, generated+auto-updated after the `dzil release` step.